src/main/java/it/gov/pagopa/pu/bff/connector/process_executions/client/IngestionFlowFileSearchClient.java

```diff
@@ -21,21 +21,27 @@ public IngestionFlowFileSearchClient(
     this.processExecutionsApisHolder = processExecutionsApisHolder;
   }
 
-  public PagedModelIngestionFlowFile getIngestionFlowFiles(IngestionFlowFileFiltersDTO ingestionFlowFileFilters, String operatorExternalId, Pageable pageable, String accessToken) {
-    return processExecutionsApisHolder.getIngestionFlowFileSearchControllerApi(accessToken)
+  public PagedModelIngestionFlowFile getIngestionFlowFiles(
+    IngestionFlowFileFiltersDTO ingestionFlowFileFilters,
+    String operatorExternalId, Pageable pageable, String accessToken) {
+    String status = ingestionFlowFileFilters.getStatus() != null ?
+      ingestionFlowFileFilters.getStatus().name()
+      : "";
```

Comment:
not empty string! null! otherwise it will be send, giving error due to the invalid value!
ps: use the ternary operator directly on the method invoke, it would be more readable there (it's not used anywere else)